packages/sit-onyx/src/components/OnyxForm/OnyxForm.core.ts

```diff
@@ -0,0 +1,101 @@
+import { computed, inject, provide, type InjectionKey, type Reactive, type Ref } from "vue";
+
+const FORM_INJECTION_KEY = Symbol() as InjectionKey<ReturnType<typeof createFormInjectionContext>>;
+
+/**
+ * Props on the `OnyxForm` component.
+ * These are injected, so that they can be used in the form child components.
+ */
+export type FormInjectedProps = {
+  disabled: boolean;
```

Comment:
```suggestion
  disabled?: boolean;
```

Disabled is optional, right?

Reply:
Not for this type, otherwise it will mess up the derived types